docs/native-debugging.md

```diff
@@ -16,8 +16,10 @@ import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import con
 
 You can display the console logs for an iOS or Android app by using the following commands in a terminal while the app is running:
 
-```sh
+```shell
+# For iOS:
 npx react-native log-ios
+# Or, for Android:
 npx react-native log-android
 ```
```

Comment:
Not strictly related to this PR, but we should order platforms alphabetically, so Android before iOS. This applies to sentences and commands too. In the other hand, I'm not sure if inline comments in there adds any value, since the platform name is a part of the command.

Reply:
>Not strictly related to this PR, but we should order platforms alphabetically, so Android before iOS. This applies to sentences and commands too.

Yep.

>I'm not sure if inline comments in there adds any value, since the platform name is a part of the command.

It does actually, removing comments _may_ convey that _BOTH_ of these commands need to be _executed_, which is _not_ the case here, at least from absolute beginners' POS👍